README.md

```diff
@@ -20,6 +20,13 @@ was written correctly and much more.
 
 ![Etcher](https://raw.githubusercontent.com/resin-io/etcher/master/screenshot.png)
 
+Supported Operating Systems
+---------------------------
+
+- GNU/Linux
```

Comment:
Maybe we should list some common distros here, like `GNU/Linux (Ubuntu, Fedora, Arch, ...)` or something, so that people won't think their Linux distro doesn't cut it?